controllers/user/Notification.js

```diff
@@ -0,0 +1,107 @@
+import moment from 'moment';
+import db from '../../models/index';
+import Util from '../../helpers/Util';
+import HttpError from '../../helpers/exceptionHandler/httpError';
+
+const { Notifications } = db;
+/**
+ * @exports Notification
+ * @class Notification
+ * @description Handles all notifications in the application
+ * */
+class Notification { /* eslint-disable */
+
+    constructor() {
+        this.message;
+        this.userId;
+    }
+
+
+    /**
+     * get all the notification of a user
+     * @async
+     * @param  {object} req - Request object
+     * @param {object} res - Response object
+     * @return {json} Returns json object
+     * @static
+     */
+    async getNotification(req, res) {
+        const { userId } = req.authData;
+        const total = await Notifications.count({ where: { userId } });
+        const pageInfo = Util.getPageInfo(req.query.page, req.query.size, total);
+        const { page, limit, offset } = pageInfo;
+        let userNotification = await Notifications.findAll({
+            limit,
+            offset,
+            where: { userId },
+            order: [
+                ['createdAt', 'DESC']
+            ]
+        });
+
+        if (userNotification.length === 0) {
```

Comment:
@blackshady , there may be no need to do this check. If the user has no notifications, sending an empty array with a `200` response is easier for the client to handle compared to having to watch out for a particular response message.